### ISSUE TITLE
"Refactor with setInterval" does nothing?

### ISSUE THREAD

**Issue by xem** [Siorki/RegPack#67]
I may do something wrong but it seems to me that this feature has no effect.

Ex: if I try to pack the code:

    a++;

I check "setInterval" and choose "a" as time var.

the packed code is:

    for(_='$++';G=/[]/.exec(_);)with(_.split(G))_=join(shift());eval(_)

Shouldn't it be something like

    for(_='$++';G=/[]/.exec(_);)with(_.split(G))_=join(shift());setInterval(_,33)

?

Or do I have to do something else ?

Thanks!

**Comment by Siorki** [comment]
Right, the current code is only looking for function patterns, not strings.
Getting it to work with your third example should not be too hard
`a=0;setInterval("a++",33);`

For the last one, I give up until I get a good ES6 string parser :)
Point made, if you want to use that feature from RegPack, don't golf the core loop into something cryptic, especially using [tagged templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_template_strings).
BTW your string `` setInterval`a++${33}b=1` `` is equivalent to `setInterval(["a++","b=1"],33)`

**Comment by xem** [comment]
Indeed, we wrote [an article](http://xem.github.io/articles/#webspeech) about it :)
It's useful for saving bytes, but not at all in the context of RegPacked code.
Anyway, no rush here, I don't need RegPack to understand setInterval with strings to deliver my demos :D 
Thanks !
